oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -81,13 +81,13 @@ protected ProcessorIdentifier queryProcessorId() {
                 }
                 continue;
             }
-            switch (splitLine[0]) {
+            switch (splitLine[0].toLowerCase()) {
```

Comment:
Make sure this is Locale independent:
```suggestion
            switch (splitLine[0].toLowerCase(Locale.ROOT)) {
```

Reply:
I know, but I see toLowerCase() being used elsewhere, so I'm going to open a new PR to clean up all use cases of toLowerCase()/toUpperCase().

Reply:
Opened https://github.com/oshi/oshi/issues/2451 to track this.